cmd/infracost/run.go

```diff
@@ -91,15 +91,26 @@ func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
 			return err
 		}
 
-		projects = append(projects, providerProjects...)
+		usageProjects := make([]*schema.Project, len(projects))
```

Comment:
@aliscott wanted to call out this slight hack due to the terragrunt stuff combined with reloading resources after syncing usage.

Do we mean to pass _all_ projects to every estimation run, and not just this local provider's? If I had a config file with 3 project roots, wouldn't the usage file for each one contain all resources or summat?

Reply:
You're right, this looks like a bug. We should only pass the provider's projects to sync usage each time.